docs/development.md

```diff
@@ -1,10 +1,11 @@
 # Running the SHIELD Dashboard locally
 
-1. `echo DEBUG=True >> .env`
+1. `echo DEBUG=true >> .env`
```

Comment:
Actually, since we already have another boolean item with `PROD_API` now which will be evaluated as a JavaScript item, I think we should stick with the lower case version. python-decouple will actually use the lower cased version as well to convert the `True` shell string to a truthy Python value [here](https://github.com/henriquebastos/python-decouple/blob/00921996aa41dd4e568fd7d91918dc9b4d167617/decouple.py#L52). So it's safe to stick with one convention (`true`).